Sources/Clappr_tvOS/Classes/Base/Player.swift

```diff
@@ -8,6 +8,7 @@ open class Player: AVPlayerViewController {
     static var hasAlreadyRegisteredPlaybacks = false
     private let baseObject = BaseObject()
     private var isChromeless: Bool { core?.options.bool(kChromeless) ?? false }
+    private var nextFocusEnvironment: UIFocusEnvironment?
```

Comment:
Making this a `UIFocusEnvironment` will require sharing the actual `UICorePlugin` reference. There is no guarantee for when it will be released after adding it to an event user info.

Can we achieve the same goal (identifying the `UIView` that required the focus) without using this kind reference (e.g.: with a simple integer or string identification) ?

Reply:
Changes made.